SharedProjects/Babylon2GLTF/GLTFExporter.Light.cs

```diff
@@ -30,6 +30,9 @@ private int AddLightExtension(ref GLTF gltf, BabylonLight babylonLight)
                 intensity = babylonLight.intensity,
             };
 
+            // Custom user properties
+            //light.extras = babylonLight.metadata;
```

Comment:
commented on purpose?

Reply:
Hi Selmar,

Good point! We commented the extras export for gltf lights as they are not loaded by the gltf loader of Babylon for now.
The export can be done by simply decommenting this line.

For the import, the question is where to perform the convertion of extra to metadata. From a conception point of view, both _ExtraAsMetadata.ts_ and _KHR_lights_punctual.ts_ extensions are required to perform the task. However the extension system doesn't provide a simple way to perform a special task when two extensions are active at the same time.

What we offer to do is copy/paste the _assignExtras method to _KHR_lights_punctual.ts_ file and assign the extras as metadata when loading the node.

@Drigax If it's ok for you we can perform a PR on the babylon repository.

Reply:
@noalak please do, that sounds like a good solution :)